### PR TITLE
Rework dashboard page #168

### PR DESCRIPTION
**Pull request by KevinLatino**

### CODE REVIEW

File: frontend/src/pages/spotnet/dashboard/dashboard.css [hunk 1]
```diff
@@ -123,11 +119,34 @@ body {
...
+  font-weight: 500;
+}
+
+.current-sum {
+  font-size: 21px;
   font-weight: 500;
 }
 
+.current-sum .green {
+  color: green;
```
Comment (whateverfw) [inline frontend/src/pages/spotnet/dashboard/dashboard.css:143]: use css variable for a color, according to design

File: frontend/src/pages/spotnet/dashboard/dashboard.css [hunk 2]
```diff
@@ -123,11 +119,34 @@ body {
...
+}
+
+.current-sum .red {
+  color: red;
```
Comment (whateverfw) [inline frontend/src/pages/spotnet/dashboard/dashboard.css:147]: use css variable for a color, according to design

File: frontend/src/pages/spotnet/dashboard/Dashboard.jsx [hunk 3]
```diff
@@ -46,6 +46,9 @@ const Dashboard = ({ walletId }) => {
       currencyIcon: UsdIcon,
     },
   ]);
+
+  const [startSum] = useState(400);
```
Comment (whateverfw) [inline frontend/src/pages/spotnet/dashboard/Dashboard.jsx:50]: default value should be 0 according to requirements

File: frontend/src/pages/spotnet/dashboard/Dashboard.jsx [hunk 4]
```diff
@@ -46,6 +46,9 @@ const Dashboard = ({ walletId }) => {
...
+  const [currentSum, setCurrentSum] = useState(200);
```
Comment (whateverfw) [inline frontend/src/pages/spotnet/dashboard/Dashboard.jsx:51]: default value should be 0 according to requirements

**Review by whateverfw — CHANGES_REQUESTED** [review]

File: frontend/src/pages/spotnet/dashboard/dashboard.css [hunk 5]
```diff
@@ -219,38 +204,65 @@ body {
...
   }
 
-  .redeem-btn {
-    padding: 0.875rem;
+  .card-custom-styles:last-child {
+    margin-left: 8px;
+  }
+
+  .icon-text-gap {
+    margin-left: 3px !important;
```
Comment (whateverfw) [inline frontend/src/pages/spotnet/dashboard/dashboard.css:260]: avoid important

File: frontend/src/pages/spotnet/dashboard/dashboard.css [hunk 6]
```diff
@@ -2,50 +2,19 @@ body {
...
-  position: relative;
-  padding: 4rem 1rem;
-  max-width: 1200px;
-  margin: 0 auto;
-}
-
-/* Background effects */
-.background-effects {
-  position: absolute;
+  margin-bottom: 12rem !important;
```
Comment (whateverfw) [inline frontend/src/pages/spotnet/dashboard/dashboard.css:8]: avoid important

**Review by whateverfw — CHANGES_REQUESTED** [review]

File: frontend/src/pages/spotnet/dashboard/Dashboard.jsx [hunk 7]
```diff
@@ -116,52 +122,84 @@ const Dashboard = ({ walletId }) => {
...
+                <div className="d-flex align-items-center mb-3">
+                  <card.currencyIcon className="card-icons rounded-circle" />
+                  <span className="ms-2 icon-text-gap text-style">{card.currencyName}</span>
+                </div>
+                <div className="d-flex align-items-center">
+                  <span className="dashboard-text-color balance-text-size">Balance:</span>
+                  <span
+                    className="ms-2 borr-text icon-text-gap text-style"
+                    style={{
+                      color: card.title === 'Borrow' ? '#FF5A5F' : '#4CAF50', // Direct color values
```
Comment (whateverfw) [inline frontend/src/pages/spotnet/dashboard/Dashboard.jsx:177]: it should be different css classnames on condiiton, and in css you should use CSS variables for colors

File: frontend/src/pages/spotnet/dashboard/Dashboard.jsx [hunk 8]
```diff
@@ -116,52 +122,84 @@ const Dashboard = ({ walletId }) => {
...
+                    }}
+                  >
+                    {card.balance}
+                  </span>
+                </div>
+                {card.title === 'Collateral & Earnings' && (
+                  <div className="sum-info">
+                    <span className="start-sum">Start sum: {startSum} $</span>
+                    <span className="current-sum">
+                      Current sum: <span style={{ color: currentSum < startSum ? '#FF5A5F' : '#4CAF50' }}>{currentSum} $</span>
```
Comment (whateverfw) [inline frontend/src/pages/spotnet/dashboard/Dashboard.jsx:187]: it should be different css classnames on condition (just like you had before), and in css you should use CSS variables for colors

File: frontend/src/pages/spotnet/dashboard/dashboard.css (hunk 2 above)
Comment (whateverfw) [reply frontend/src/pages/spotnet/dashboard/dashboard.css]: @KevinLatino this version could've been kept, just with a change of `red` to corresponding CSS variable

File: frontend/src/pages/spotnet/dashboard/dashboard.css (hunk 1 above)
Comment (whateverfw) [reply frontend/src/pages/spotnet/dashboard/dashboard.css]: @KevinLatino this version could've been kept, just with a change of `red` to corresponding CSS variable

File: frontend/src/pages/spotnet/dashboard/dashboard.css [hunk 9]
```diff
@@ -152,101 +151,297 @@ body {
...
+}
+
+.current-sum {
+  font-size: 21px;
+  font-weight: 500;
+  color: white; /* "Current sum" text in white */
+}
+
+.current-sum .current-sum-green {
+  color: #4CAF50; /* Green for positive values */
```
Comment (whateverfw) [inline frontend/src/pages/spotnet/dashboard/dashboard.css:197]: this class should be used in a component, and also color should be CSS variable

File: frontend/src/pages/spotnet/dashboard/dashboard.css [hunk 10]
```diff
@@ -152,101 +151,297 @@ body {
...
+}
+
+.current-sum .current-sum-red {
+  color: #FF5A5F; /* Red for negative values */
```
Comment (whateverfw) [inline frontend/src/pages/spotnet/dashboard/dashboard.css:201]: this class should be used in a component, and also color should be CSS variable

**Review by whateverfw — CHANGES_REQUESTED** [review]

**Review by whateverfw — APPROVED** [review]
Looks good to me, thanks.

**Review by djeck1432 — APPROVED** [review]
Looks good to me

**Pull request merged**